app/pandas.ipynb

```diff
@@ -333,114 +764,275 @@
+    }
+   ],
+   "execution_count": 59
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# use ** operator here"
-   ],
+   "source": "s1 ** s2",
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-07T15:24:22.258211Z",
+     "start_time": "2024-11-07T15:24:22.247068Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0          1\n",
+       "1        128\n",
+       "2       6561\n",
+       "3     262144\n",
+       "4    9765625\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 60,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 60
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# use // operator here"
-   ],
+   "source": "s1 // s2",
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-07T15:24:24.036777Z",
+     "start_time": "2024-11-07T15:24:24.027263Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    0\n",
+       "1    0\n",
+       "2    0\n",
+       "3    0\n",
+       "4    0\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 61,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 61
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# use % operator here"
-   ],
+   "source": "s1 % s2",
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-07T15:24:29.218989Z",
+     "start_time": "2024-11-07T15:24:29.207890Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    1\n",
+       "1    2\n",
+       "2    3\n",
+       "3    4\n",
+       "4    5\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 62,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 62
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# filter s1 here"
-   ],
+   "source": "s1[s1 % 2 == 0]",
    "metadata": {
-    "collapsed": false
-   }
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2024-11-07T15:24:38.744953Z",
+     "start_time": "2024-11-07T15:24:38.726301Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1    2\n",
+       "3    4\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 63,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 63
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# filter s2 here"
-   ],
+   "source": "s1[s1 % 2 == 1]",
```

Comment:
In Task 5.2, you are required to filter `s2` to print only odd numbers, but the code provided filters `s1` for odd numbers instead. You should apply the filter `s2[s2 % 2 == 1]` to `s2` to get the correct result.